recipe/meta.yaml

```diff
@@ -21,9 +21,9 @@ requirements:
   run:
     - python
     - ipython >=4.0.0
-    - ipykernel >=4.5.1
-    - traitlets >=4.3.1
-    - widgetsnbextension >=2.0.0
+    - ipykernel >=4.2.2
+    - traitlets >=4.2.1
+    - widgetsnbextension >=1.2.6, ==1.*
```

Comment:
> ok, that was from the PEP 440 documentation.

IIRC, conda does not understand all of the PEP 440 version numbers (for example, I don't think it understands `~`).